{"uuid": "05d249c3-3171-47b9-bd7d-3b2ef9f27816", "children": ["3214e319-4b92-4af2-9023-636c27f39472"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=138.0.7204.93)\nStacktrace:\n0   chromedriver                        0x0000000100540ea4 cxxbridge1$str$ptr + 2722840\n1   chromedriver                        0x0000000100538dac cxxbridge1$str$ptr + 2689824\n2   chromedriver                        0x000000010008a3ec cxxbridge1$string$len + 90648\n3   chromedriver                        0x0000000100064144 chromedriver + 131396\n4   chromedriver                        0x00000001000f960c cxxbridge1$string$len + 545848\n5   chromedriver                        0x0000000100112454 cxxbridge1$string$len + 647808\n6   chromedriver                        0x00000001000c5834 cxxbridge1$string$len + 333408\n7   chromedriver                        0x0000000100503fc0 cxxbridge1$str$ptr + 2473268\n8   chromedriver                        0x000000010050722c cxxbridge1$str$ptr + 2486176\n9   chromedriver                        0x00000001004e5a08 cxxbridge1$str$ptr + 2348924\n10  chromedriver                        0x0000000100507ae8 cxxbridge1$str$ptr + 2488412\n11  chromedriver                        0x00000001004d6a98 cxxbridge1$str$ptr + 2287628\n12  chromedriver                        0x00000001005279d8 cxxbridge1$str$ptr + 2619212\n13  chromedriver                        0x0000000100527b64 cxxbridge1$str$ptr + 2619608\n14  chromedriver                        0x00000001005389e8 cxxbridge1$str$ptr + 2688860\n15  libsystem_pthread.dylib             0x00000001967c5f94 _pthread_start + 136\n16  libsystem_pthread.dylib             0x00000001967c0d34 thread_start + 8\n\n", "trace": "  File \"/Users/sumit/PycharmProjects/PartnerPortalUI/.venv/lib/python3.9/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n  File \"/Users/sumit/PycharmProjects/PartnerPortalUI/.venv/lib/python3.9/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/Users/sumit/PycharmProjects/PartnerPortalUI/.venv/lib/python3.9/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/Users/sumit/PycharmProjects/PartnerPortalUI/.venv/lib/python3.9/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/sumit/PycharmProjects/PartnerPortalUI/.venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/sumit/PycharmProjects/PartnerPortalUI/.venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Users/sumit/PycharmProjects/PartnerPortalUI/conftest.py\", line 73, in driver\n    driver.maximize_window()\n  File \"/Users/sumit/PycharmProjects/PartnerPortalUI/.venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 641, in maximize_window\n    self.execute(command, None)\n  File \"/Users/sumit/PycharmProjects/PartnerPortalUI/.venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 448, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/sumit/PycharmProjects/PartnerPortalUI/.venv/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1752140836806, "stop": 1752140840406}], "start": 1752140836806, "stop": 1752140840569}